STATIC_ANALYSIS.md

```
@@ -0,0 +1,1550 @@
+```
+
+### 5.2 Return Type Inconsistencies
+
+| Function | Declared Return | Actual Return |
+|----------|-----------------|---------------|
+| `de_code_value()` | none | `string` |
+| `en_code_value()` | none | `string` |
+| `get_from_cookies()` | none | `string` |
+| `create_jwt()` | `string` | `string` (correct) |
+| `verify_jwt()` | none | `array [string, string]` |
+| `get_access_from_dbs()` | none | `array|null` |
```

markdownlint-cli2 (0.20.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)